src/app.js

```diff
@@ -1,8 +1,13 @@
+require('dotenv').config({path:'./env'})
```

Comment:
do we also need this line here? as we already had requested for dotenv in the server.js file...

Reply:
I think you are right about this Olga, i will remove it..